src/Stratis.Features.Unity3dApi/NFTTransferIndexer.cs

```diff
@@ -89,10 +91,17 @@ public void Initialize()
             this.logger.LogDebug("NFTTransferIndexer initialized.");
         }
 
+        private int GetWatchFromHeight()
+        {
+            int watchFromHeight = this.network.IsTest() ? 3000000 : 3400000;
```

Comment:
Perhaps define constants for these.

Reply:
I'll remove it entirely once we have ability to get deployment height by contract address